subprojects/files/src/test/groovy/org/gradle/api/internal/file/collections/DirectoryFileTreeTest.groovy

```diff
@@ -89,6 +80,7 @@ class DefaultDirectoryWalkerTest extends AbstractProjectBuilderSpec {
 
         Test that the files are really walked breadth first
      */
+
```

Comment:
Why did you add this new line?

Reply:
Probably automatic formatting during revert

Reply:
Done